rollup.config.mjs

```diff
@@ -0,0 +1,55 @@
+import { babel } from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import nodeResolve from '@rollup/plugin-node-resolve';
+import builtins from 'rollup-plugin-node-builtins';
+import glob from 'glob';
+import * as path from "path"
+
+const entrypoints = {}
+
+glob
+  .sync("./src/**/*.js")
+  .forEach((file) => {
+    const {dir, name} = path.parse(path.relative("./src", file))
+    const shortPath = path.join(dir, name)
+    entrypoints[shortPath] = file
+  })
+
+const plugins = [
+  builtins(),
+  nodeResolve(),
+  commonjs({
+	  include: /\/node_modules\//,
+	  esmExternals: false,
+	  requireReturnsDefault: 'namespace',
+  }),
+  babel({
+    babelHelpers: "inline",
+    comments: false,
+  }),
+]
+
+// put shared chunks into lib/chunks/*
+const chunkFileNames = (chunkInfo) => 'chunks/[name].[hash].js'
+
+/** @type {import("rollup").RollupOptions[]} */
+export default [
+  {
+    input: entrypoints,
+    output: {
+      dir: 'lib/esm',
+      format: 'es',
+      chunkFileNames,
```

Comment:
@KonnorRogers, I had to add [`intro: 'var global = typeof self !== undefined ? self : this;'`](https://github.com/rollup/rollup-plugin-commonjs/issues/6#issuecomment-526447762) to get rid of `Uncaught ReferenceError: global is not defined at elementRoleMap.js:1446:1` when using `import { roles } from "./node_modules/aria-query/lib/esm/index.js";`.

Reply:
Added! Thanks! I hit the same issue as well.